src/js/views/globals/app-nav/app-nav_views.js

```diff
@@ -31,34 +31,61 @@ const MainNavDroplist = Droplist.extend({
         <img class="app-nav__header-logo" src="/rwell-logo.svg" />
       {{else}}
         <div class="u-text--overflow">
-          <h2 class="app-nav__header-title u-text--overflow">{{ orgName }}</h2>
+          <h2 class="app-nav__header-title u-text--overflow">{{ workspaceName }}</h2>
           <span class="app-nav__header-arrow">{{far "angle-down"}}</span>
         </div>
         <div class="u-text--overflow">{{ userName }}</div>
       {{/if}}
     `,
     templateContext() {
       const currentUser = Radio.request('bootstrap', 'currentUser');
-      const currentOrg = Radio.request('bootstrap', 'organization');
+      const currentWorkspace = Radio.request('bootstrap', 'currentWorkspace');
 
       return {
         userName: currentUser.get('name'),
-        orgName: currentOrg.get('name'),
+        workspaceName: currentWorkspace.get('name'),
         isMinimized: this.getOption('state').get('isMinimized'),
       };
     },
   },
   picklistOptions() {
     return {
       className: 'picklist app-nav__picklist',
-      itemClassName: 'flex flex-align-center app-nav__picklist-item',
-      lists: [{
-        collection: this.collection,
-        itemTemplate: hbs`
-          {{fa icon.type icon.icon classes=icon.classes~}}
-          <span>{{formatMessage text}}</span>
-        `,
-      }],
+      template: hbs`
+        <div class="app-nav__picklist-heading">{{ @intl.globals.appNav.appNavViews.mainNavDroplist.organizationHeading }}</div>
+        <div class="app-nav__picklist-workspace-name">{{ name }}</div>
+        <ul class="flex-region picklist__scroll js-picklist-scroll"></ul>
```

Comment:
What if we put the heading text for workspaces above the `ul` here and then we could avoid modifying the picklist component.  It might be inevitable at some point, but for now it's an edge case.. I know it'll scroll a little differently, but that may not matter.